src/utils.js

```diff
@@ -44,7 +44,8 @@ const VALUE_EXTRACT_MAP = {
     'STRING': (value) => value.stringValue,
     'ENUMERATED': (value) => value.stringValue,
     'TIMESTAMP': (value) => value.stringValue,
-    'BOOLEAN': (value) => value.booleanValue
+    'BOOLEAN': (value) => value.booleanValue,
+    'BINARY': (value) => value.binaryValue
```

Comment:
Intriguing...I wonder what this looks like in JSON? Base64?

Reply:
Yep, this comes in as a Base64 string. The YAMCS explorer actually decodes this and displays it as hex separated into 2-byte chunks. I attempted to do the same here, but for the sake of simplicity of a first-pass, went with just displaying the Base64 string.

Reply:
Cool! And yeah, Base64 is fine for now. Tackling binary as hex is out of scope.